src/planscape/metrics/tests/test_views.py

```diff
@@ -0,0 +1,132 @@
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(
+            url,
+            {"organization": self.organization.id},
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(response.data["results"][0]["name"], self.metric.name)
+
+    def test_filter_by_project(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(url, {"project": self.project.id})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(response.data["results"][0]["name"], self.metric.name)
+
+    def test_filter_by_category(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(url, {"category": self.category.id})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(response.data["results"][0]["name"], self.metric.name)
+
+    def test_filter_by_capabilities(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(
+            url,
+            {"capabilities": MetricCapabilities.MAP_VIEW},
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(response.data["results"][0]["name"], self.metric.name)
+
+    def test_filter_by_name_exact(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(url, {"name": "Test Metric"})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(response.data["results"][0]["name"], self.metric.name)
+
+    def test_filter_by_name_contains(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(url, {"name__contains": "Test"})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(response.data["results"][0]["name"], self.metric.name)
+
+    def test_filter_by_display_name_exact(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(url, {"display_name": "Test Metric Display"})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data.get("results")), 1)
+        self.assertEqual(
+            response.data["results"][0]["display_name"],
+            self.metric.display_name,
+        )
+
+    def test_filter_by_display_name_contains(self):
+        url = reverse("metrics:metrics-list")
+        response = self.client.get(url, {"display_name__contains": "Metric"})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Do we want to check for result count in these filter tests?

Reply:
Nah, if we're already counting, that's fine. For some of the trickier complicated filters, it's nice to guard against breaking changes, but I think this is fine.